atest/testdata/standard_libraries/collections/list.robot

```diff
@@ -331,6 +331,11 @@ Lists Should Be Equal With Named Indices As Dictionary With Too Few Values
     ${names} =    Create Dictionary    0=a    2=c
     Lists Should Be Equal    ${L3}    ${L3B}    names=${names}
 
+Lists Should Be Equal Ignore Order    
+    ${list1} =    Create List    A    B    C    D
+    ${list2} =    Create List    D    B    C	A
```

Comment:
Somehow formatting of this shown differently but I just checked checked in file here and it 
shows correct. 

![image](https://user-images.githubusercontent.com/33742811/74623859-3ffbb600-516c-11ea-86ac-902a73a13862.png)

Reply:
You probably have a tab at least between `C` and `A`.